templates/saas.amazonaws.com.yaml

```diff
@@ -63,17 +88,36 @@ identities:
     dns_pattern: .*email.*.amazonaws.com
     dst_port: 443
   pushdown: []
+- node:
+    name: pinpoint
+    dns_pattern: pinpoint.*amazonaws.com
+    dst_port: 443
+  pushdown: []
+- node:
+    name: pricing
+    dns_pattern: .*pricing.*.amazonaws.com
+    dst_port: 443
+  pushdown: []
 - node:
     name: rabbitmq
     dns_pattern: .*mq.*.amazonaws.com
     dst_port: 443
   pushdown: []
+- node:
+    name: rds
+    dns_pattern: .*rds.*.amazonaws.com
+    dst_port: 5432
+  pushdown: []
+- node:
+    name: route53
+    dns_pattern: .*route53.*.amazonaws.com
+    dst_port: 443
+  pushdown: []
 - node:
     name: s3
     dns_pattern: .*s3.*.amazonaws.com
     dst_port: 443
-  pushdown:
-  - dns_pattern
```

Comment:
did you intend to remove the pushdown?

Reply:
yes, it wasn't specific enough (didn't have dns pattern listed)